### DISCUSSION TITLE
Toolbar Position: Sidebar

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#98] [Ideas]
[On Reddit](https://www.reddit.com/r/ObsidianMD/comments/1dv3mmz/help_needed_add_buttons_to_obsidian_sidebar/), /u/krifire was looking for a way to show toolbar items in a separate view:

> I’d like to add programmed macros or buttons in the sidebar. I already have a calendar plugin there and believe I can add another tab with buttons.

Screenshot from a different note-taking app:

![image](https://github.com/chrisgurney/obsidian-note-toolbar/assets/17337703/db63c640-c7e3-48d5-86c3-26d8876685dd)

### Thoughts

1. Would there be interest from others on such a style of toolbar?

  - This would keep this style of toolbar out of the way, allowing the user to bring it into view only when needed.

2. Can anybody point me to any community plugin that shows note information in a sidebar view? I'm aware of several core plugins (e.g., Backlinks) that do this, but would love to see some code.

  - Found this: [Calendar plugin source](https://github.com/liamcain/obsidian-calendar-plugin/blob/master/src/main.ts)

**Comment by MrComputron** [comment]
Could this not be accomplished by pinning a note to the sidebar with any number of in line toolbars?

Unless the idea is to position general purpose tools off to the side that will affect the active note which might not work if it's another note entirely.

I like the idea of having a "toolbox" off to the side if it's possible.

**Reply by chrisgurney** [reply]
@MrComputron Yes it entirely depends on what the tools are. I think your idea could work for navigation (assuming you open the items in a new tab) for example.
